<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlayerWindow</class>
 <widget class="QWidget" name="AddPlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addPlayerFirstName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addPlayerLastName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addPlayerAddress"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addPlayerButton">
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
